packages/gcm/lib/gcm.dart

```diff
@@ -0,0 +1,10 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Service exposed to Flutter apps that implements a subset of the GCM API.
+///
+/// This library will probably be moved into a separate package eventually.
```

Comment:
Agreed. Our plan is to put them in the main repo first to see what is needed, then move them to a separate repo.